components/eamxx/src/share/util/scream_vertical_interpolation_impl.hpp

```diff
@@ -46,8 +46,8 @@ void apply_interpolation_impl_1d(
   auto mask   = Kokkos::subview(mask_col,  Kokkos::pair<int,int>(0,num_tgt_packs));
 
   // The input/output data and x_src/x_tgt data should match in the appropriate size, respectively.
-  EKAT_REQUIRE_MSG(x_tgt.size() == output.size(),"Error! vertical_interpolation::perform_checks " + std::to_string(x_tgt.size()) + " != " + std::to_string(output.size()) + ".");
-  EKAT_REQUIRE_MSG(x_src.size() == input.size(), "Error! vertical_interpolation::perform_checks " + std::to_string(x_src.size()) + " != " + std::to_string(input.size()) + ".");
+  EKAT_KERNEL_REQUIRE_MSG(x_tgt.size() == output.size(),"Error! vertical_interpolation::perform_checks " + std::to_string(x_tgt.size()) + " != " + std::to_string(output.size()) + ".");
```

Comment:
I don't think this will work. The KERNEL assertions require C style strings, not std::string. Given that you can't allocate a C string here to create the message (e.g., via sprintf), your only option is to not print any info regarding the data, just a plain error message string.

Reply:
Huh, that's probably why the fail in the AT.  Interestingly I kept that because it was working on my weaver build.  Oh well.  I'll go ahead and take away the message, it isn't strictly necessary anyways.